notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    import openvino
</pre><p><code>import openvino as ov</code> is a conventional  way</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Changed